Content.Shared/Access/Systems/IdExaminableSystem.cs

```diff
@@ -27,7 +27,9 @@ private void OnGetExamineVerbs(EntityUid uid, IdExaminableComponent component, G
         {
             Act = () =>
             {
-                var markup = FormattedMessage.FromMarkup(info);
+                if (!FormattedMessage.TryFromMarkup(info, out var markup))
```

Comment:
I'd probably rather this threw than silently failing, so this should use `FromMarkupOrThrow` instead.

Reply:
Done :heart: